pkgs/applications/graphics/qview/default.nix

```diff
@@ -0,0 +1,22 @@
+{stdenv, fetchFromGitHub, qmake}:
+stdenv.mkDerivation rec {
+  name = "qview-${version}";
```

Comment:
Please use `pname = "qview";` and then `name` will be then care of automatically for you.